nvflare/private/fed/server/server_runner.py

```diff
@@ -335,8 +336,8 @@ def _try_to_get_task(self, client, fl_ctx, timeout=1.0, retry_interval=0.005):
 
                 time.sleep(retry_interval)
 
-            # ask client to retry
-            return "", "", None
+        # ask client to retry
+        return "", "", None
```

Comment:
Is this intended?

Reply:
Yes, this results to SpecialTaskName.TRY_AGAIN